ChameleonForms/FieldGenerators/Handlers/FieldGeneratorHandler.cs

```diff
@@ -189,7 +252,14 @@ protected static IHtmlString GetSelectListHtml(IEnumerable<SelectListItem> selec
                     return fieldGenerator.Template.RadioOrCheckboxList(list, isCheckbox: HasMultipleValues(fieldGenerator));
                 case FieldDisplayType.DropDown:
                 case FieldDisplayType.Default:
-                    return HasMultipleValues(fieldGenerator)
+                    if (HasMultipleEnumValues(fieldGenerator))
+                    {
+                        var attrs = new HtmlAttributes(fieldConfiguration.HtmlAttributes);
+                        AdjustHtmlForModelState(attrs, fieldGenerator);
+                        return HtmlCreator.BuildSelect(GetFieldName(fieldGenerator), selectList, multiple: true, htmlAttributes: attrs);

```

Comment:
Couldn't use `ListBoxFor` since it threw a hissy fit if the value wasn't Enumerable

Eventually we should just use HtmlCreator for everything and move away from the built-in MVC stuff